### ISSUE TITLE
Is this official repository for the MV3 extension?

### ISSUE THREAD

**Issue by laserjobs** [Windscribe/browser-extension-mv3#1]
This version says 1.1.0 and version 1.2.0 Updated on August 12, 2024 at
https://chromewebstore.google.com/detail/windscribe-experimental-m/fgcngeihbacfndglmmmdkkdlhgndkaaf

**Comment by ahmed-rafi-ullah** [comment]
yes it is the official repo for the mv3 extension and the experimental mv3 extension
